### DISCUSSION TITLE
Docker (HTTP code 500) Server Error - Ports are not available

### DISCUSSION THREAD

**Discussion by shpirtim** [ptrumpis/snap-camera-server#127] [Q&A] [answered]
In docker: 
"(HTTP code 500) server error - Ports are not available: exposing port TCP 0.0.0.0:80 -> 0.0.0.0:0: listen tcp 0.0.0.0:80: bind: An attempt was made to access a socket in a way forbidden by its access permissions."

Worked perfectly fine until this port stopped working maybe two weeks ago.

**Comment by ptrumpis** [comment, marked as answer]
This error occurs if a port is already in use by another process or docker container.

You can change the port to a different number in your .env file:
https://github.com/ptrumpis/snap-camera-server/blob/main/example.env#L13
